src/main/java/com/faforever/api/data/converter/MapParamsConverter.java

```diff
@@ -0,0 +1,50 @@
+package com.faforever.api.data.converter;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import lombok.extern.slf4j.Slf4j;
+
+import javax.persistence.AttributeConverter;
+import javax.persistence.Converter;
+import java.io.IOException;
+import java.util.Map;
+
+@Slf4j
+@Converter
+public class MapParamsConverter implements AttributeConverter<Map<String, Object>, String> {
```

Comment:
This is not a MapParamsConverter. This is a reusable JsonConverter.

Reply:
Done